infrastructure/code/snyk_code_http_client.go

```diff
@@ -499,8 +499,8 @@ func (s *SnykCodeHTTPClient) RunAutofix(ctx context.Context, options AutofixOpti
 		logger.Err(err).Msg(failedToObtainRequestIdString + err.Error())
 		return AutofixResponse{}, err
 	}
-	logger.Debug().Msg("API: Retrieving autofix for bundle")
-	defer logger.Debug().Msg("API: Retrieving autofix done")
+	logger.Info().Msg("API: Retrieving autofix for bundle")
```

Comment:
This is executed in a loop, thus would spam the logs. Could you maybe add a log message that is displayed only once when on Info level?

Reply:
@bastiandoetsch Basti, I'd need your advice on it. First idea is to put the single log outside of the ticking loop here https://github.com/snyk/snyk-ls/blob/c47764c304734a01cc5e2798c4525b190839c716/infrastructure/code/autofix.go#L85 but then I'd need to calculate `requestId` also outside, and pass `requestId` to https://github.com/snyk/snyk-ls/blob/c47764c304734a01cc5e2798c4525b190839c716/infrastructure/code/autofix.go#L99, is that fine with you?